Police-Data-Accessibility-Project/prod-to-dev-migration
# Create relations_access_permission logic.

#### Fixes

* Sets up database schema changes for https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/391

#### Description

* Adds `relation_column` and `column_permission` tables to database
* Creates logic for populating above tables with information from configuration csvs. 
* Creates helper module for quickly setting permissions for the given configuration csvs.

#### Testing

* Run Database refresh in Automation.pdap.io
* Confirm scripts run as expected, and tables are properly populated in database.

#### Performance

* Adds additional time to database refresh process, due to the creation of new schemas and the addition of new data.

#### Docs

* Not applicable.

#### Breaking Changes

* Not applicable.
